tests/test_websocket/test_request.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+
+from lbz.websocket import ActionType, WebSocketRequest
+
+
+class TestWebSocketRequest:
+    def test__decode_base64_bytes(self) -> None:
+        encoded = b"asdasdsd"
+        output = WebSocketRequest._decode_base64(encoded)  # pylint: disable=protected-access
+        assert output == b"j\xc7Z\xb1\xdb\x1d"
+
+    def test__decode_base64_str(self) -> None:
+        encoded = "asdasdsd"
+        output = WebSocketRequest._decode_base64(encoded)  # pylint: disable=protected-access
+        assert output == b"j\xc7Z\xb1\xdb\x1d"
+
```

Comment:
You should not test protected methods. This is used by the `raw_body` property and is tested there:
```suggestion
```